docker-compose.yml

```diff
@@ -39,16 +39,18 @@ services:
       - "6379"
 
   elastic:
-    image: docker.elastic.co/elasticsearch/elasticsearch:6.8.2
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.10.0
+    container_name: elastic
     environment:
       - network.host=0.0.0.0
       - http.cors.enabled=true
       - http.cors.allow-origin=*
       - http.max_content_length=10mb
       - rest.action.multi.allow_explicit_index=false
       - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - discovery.type=single-node
```

Comment:
I noticed that elastic shuts down without this line locally, but my guess is RC/prod isn't single-node.  Probably doesn't matter in terms of how the code works though, right?  I googled this setting and it doesn't seem to require any other changes.

Reply:
This was suggested for local (and as you noticed it's unhappy without), but prod is multiple nodes - I'm pretty sure @Ardiea handles that elsewhere? Or do I need to create yet another yaml file?

Reply:
No this won't affect anything outside of the local dev envs.